cat-launcher/src-tauri/src/variants/commands.rs

```diff
@@ -0,0 +1,21 @@
+use tauri::{command, State};
+use crate::variants::GameVariant;
+use crate::variants::repository::sqlite_game_variant_order_repository::SqliteGameVariantOrderRepository;
+use crate::variants::GameVariantOrderRepository;
+
+#[derive(thiserror::Error, Debug, serde::Serialize, strum::IntoStaticStr)]
+pub enum UpdateGameVariantOrderCommandError {
+    #[error("failed to update game variant order")]
+    Update(String),
+}
+
+#[command]
+pub async fn update_game_variant_order(
+    variants: Vec<GameVariant>,
+    game_variant_order_repository: State<'_, SqliteGameVariantOrderRepository>,
+) -> Result<(), UpdateGameVariantOrderCommandError> {
+    game_variant_order_repository
```

Comment:
A command should not directly use a repository. Create a business logic function and use it in the command.